README.md

```diff
@@ -533,6 +533,122 @@ No aggregate operation is needed and it should be set `raw_values`. The `valueKe
+tracking module to identify which set of fields can swap values and still be considered as the same connection.
+
+The configuration example below defines a bidirectional setting. So flow-logs that have the values of `SrcAddr` and `SrcPort` 
+swapped with `DstAddr` and `DstPort` are grouped together as long as they have the same `Proto` field.
+For example, the following first 2 flow-logs are grouped together into the same connection.
+While the third flow-log forms a new connection (because its `Proto` field differs from the first 2).
+```json
+{"SrcAddr":"10.0.0.1", "SrcPort":1234, "DstAddr":"10.0.0.2", "DstPort":80, "Proto":6, "Bytes":100, "TimeReceived": 1661430100}
+{"SrcAddr":"10.0.0.2", "SrcPort":80, "DstAddr":"10.0.0.1", "DstPort":1234, "Proto":6, "Bytes":200, "TimeReceived": 1661430200}
+{"SrcAddr":"10.0.0.1", "SrcPort":1234, "DstAddr":"10.0.0.2", "DstPort":80, "Proto":17, "Bytes":300, "TimeReceived": 1661430300}
+```
+
+A typical configuration might look like:
+```yaml
+parameters:
+- name: extract_conntrack
+  extract:
+    type: conntrack
+    conntrack:
+      endConnectionTimeout: 30s
+      updateConnectionInterval: 10s
+      keyDefinition:
+        fieldGroups:
+        - name: src
+          fields:
+          - SrcIP
+          - SrcPort
+        - name: dst
+          fields:
+          - DstIP
+          - DstPort
+        - name: protocol
+          fields:
+          - Proto
+        hash:
+          fieldGroupRefs:
+          - protocol
+          fieldGroupARef: src
+          fieldGroupBRef: dst
+      outputRecordTypes:
+      - newConnection
+      - endConnection
+      - updateConnection
+      - flowLog
+      outputFields:
+      - name: Bytes_total
+        operation: sum
+        input: Bytes
+      - name: Bytes
+        operation: sum
+        splitAB: true
+      - name: numFlowLogs
+        operation: count
+      - name: TimeFlowStart
+        operation: min
+        input: TimeReceived
+      - name: TimeFlowEnd
+        operation: max
+        input: TimeReceived
+```
+
+A possible output would look like:
+```json
+{
+    "_RecordType": "endConnection",
+    "_HashId": "3e8ba98164baecaf",
+    "SrcAddr": "10.0.0.1",
+    "SrcPort": 1234,
+    "DstAddr": "10.0.0.2",
+    "DstPort": 80,
+    "Proto": 6,
+    "Bytes_AB": 100,
+    "Bytes_BA": 200,
+    "Bytes_total": 300,
+    "numFlowLogs": 2,
+    "TimeFlowStart": 1661430100,
+    "TimeFlowEnd": 1661430200
+}
+
+{
+    "_RecordType": "flowLog",
+    "_HashId": "bca4c313a1ad1b1c",
+    "SrcAddr": "10.0.0.1",
+    "SrcPort": 1234,
+    "DstAddr": "10.0.0.2",
+    "DstPort": 80,
+    "Proto": 17,
+    "Bytes": 300,
+    "TimeReceived": 1661430300
+}
+```
+
+Notice that all output records contain `_RecordType` and `_HashId` fields.
+Output fields that set `splitAB: true` (like in `Bytes`) are split into 2 fields `Bytes_AB` and `Bytes_BA` which 
+aggregate values separately based on direction A->B and B->A respectively.
```

Comment:
and also adds a `XX_total` field ?

Is there other fields than `Bytes` ?

Reply:
> and also adds a `XX_total` field ?

The `Bytes_total` field is created because it was specified explicitly in the [config snippet.](https://github.com/netobserv/flowlogs-pipeline/pull/291/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R603-R605)
It's not auto-generated by the conntrack module. I created an issue to discuss this: https://github.com/netobserv/flowlogs-pipeline/issues/280



> Is there other fields than `Bytes` ?

In this config snippet, `Bytes` is the only field that is set with `splitAB: true`.
For all the other fields, the default is set to `false`.
I'll add a sentence for the default case.